io.otsaloma.nfoview.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.otsaloma.nfoview
+runtime: org.gnome.Platform
+runtime-version: 3.30
+sdk: org.gnome.Sdk
+command: nfoview
+rename-appdata-file: nfoview.appdata.xml
```

Comment:
Should be renamed upstream. Same for the icon, desktop file. 
See the specs:
https://standards.freedesktop.org/desktop-entry-spec/latest/ar01s02.html
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html